src/QueryM/Ogmios.purs

```diff
@@ -154,18 +193,174 @@ instance DecodeAeson SubmitTxR where
     \o -> getField o "SubmitSuccess" >>= flip getField "txId" >>= hexToByteArray
       >>> maybe (Left (TypeMismatch "Expected hexstring")) (pure <<< wrap)
 
+---------------- SYSTEM START QUERY RESPONSE & PARSING
+newtype SystemStartQR = SystemStartQR String
+
+derive instance Generic SystemStartQR _
+derive instance Newtype SystemStartQR _
+derive newtype instance DecodeAeson SystemStartQR
+
+instance Show SystemStartQR where
+  show = genericShow
+
+---------------- CURRENT EPOCH QUERY RESPONSE & PARSING
+newtype CurrentEpochQR = CurrentEpochQR BigInt
+
+derive instance Generic CurrentEpochQR _
+derive instance Newtype CurrentEpochQR _
+derive newtype instance DecodeAeson CurrentEpochQR
+
+instance Show CurrentEpochQR where
+  show = genericShow
+
+---------------- ERA SUMMARY QUERY RESPONSE & PARSING
+
+newtype EraSummariesQR = EraSummariesQR (Array EraSummary)
+
+derive instance Generic EraSummariesQR _
+derive instance Newtype EraSummariesQR _
+
+instance Show EraSummariesQR where
+  show = genericShow
+
+instance DecodeAeson EraSummariesQR where
+  decodeAeson = aesonArray (map wrap <<< traverse decodeAeson)
+
+newtype EraSummary = EraSummary
+  { start :: EraSummaryTime
+  , end :: Maybe EraSummaryTime
+  , parameters :: EraSummaryParameters
+  }
+
+derive instance Generic EraSummary _
+derive instance Newtype EraSummary _
+
+instance Show EraSummary where
+  show = genericShow
+
+instance DecodeAeson EraSummary where
+  decodeAeson = aesonObject $ \o -> do
+    start <- getField o "start"
+    -- The field "end" is required by Ogmios API, but it can optionally return
+    -- Null, so we want to fail if the field is absent but make Null value
+    -- acceptable in presence of the field (hence why "end" is wrapped in
+    -- `Maybe`).
+    end' <- getField o "end"
+    end <- if isNull end' then pure Nothing else Just <$> decodeAeson end'
+    parameters <- getField o "parameters"
+    pure $ wrap { start, end, parameters }
+
+newtype EraSummaryTime = EraSummaryTime
+  { time :: RelativeTime -- 0-18446744073709552000, A time in seconds relative to
```

Comment:
you can be more specific here:
"   [...] The time is relative to the start time of the network."
"